pkg/db/db.go

```diff
@@ -0,0 +1,48 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	_ "modernc.org/sqlite"
+)
+
+var db *sql.DB
+
+const (
+	schema = `
+	CREATE TABLE scheduler (
+    	id INTEGER PRIMARY KEY AUTOINCREMENT,
+    	date CHAR(8) NOT NULL DEFAULT '',
+    	title VARCHAR(255) NOT NULL DEFAULT '',
+    	comment TEXT NOT NULL DEFAULT '',
+   		repeat VARCHAR(128) NOT NULL DEFAULT ''
+	);
+	CREATE INDEX idx_date ON scheduler (date);
+	`
+)
+
+func Init(dbFile string) error {
+	_, err := os.Stat(dbFile)
+
+	install := os.IsNotExist(err)
+
+	// Открываем базу данных
+	db, err = sql.Open("sqlite", dbFile)
+	if err != nil {
+		return fmt.Errorf("failed to open database: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("failed to open database: %w", err)
```

Reply:
Нужно везде поправить